trace_decoder/src/processed_block_trace.rs

```diff
@@ -271,11 +288,13 @@ impl TxnInfo {
                     ContractCodeUsage::Read(c_hash) => {
                         contract_code_accessed
                             .entry(c_hash)
-                            .or_insert_with(|| code_hash_resolve_f(&c_hash));
+                            .or_insert_with(|| code_hash_resolve.resolve_code_hash(&c_hash));
```

Comment:
not a big fan of `code_hash_resolve.resolve_code_hash`, but I guess that's how it is 🤷🏼

Reply:
We could maybe change the argument to `code_hash_resolver: &mut CodeHashResolving<F>,` that nomenclature is used elsewhere and it makes it _slightly_ less strange, the emphasis being very much on slightly. But I can't say I care that much tbh.

Reply:
Yeah it's an improvement. Maybe I should also rename the method `resolve_code_hash` --> `resolve`? Then we have `code_hash_resolver.resolve` which is I think I lot nicer.

Reply:
Yeah good call. Let's do that and I think we're good to go